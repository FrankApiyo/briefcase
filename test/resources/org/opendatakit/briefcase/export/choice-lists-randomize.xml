<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns="http://www.w3.org/2002/xforms">
  <h:head>
    <h:title>Randomize choice list</h:title>
    <model>
      <instance>
        <data id="choice-lists-randomize">
          <select_multiple/>
          <meta>
            <instanceID/>
          </meta>
        </data>
      </instance>
      <instance id="choices">
        <root>
          <item>
            <name>a</name>
            <label>A</label>
          </item>
          <item>
            <name>b</name>
            <label>B</label>
          </item>
          <item>
            <name>c</name>
            <label>C</label>
          </item>
          <item>
            <name>d</name>
            <label>D</label>
          </item>
          <item>
            <name>e</name>
            <label>E</label>
          </item>
        </root>
      </instance>
      <bind nodeset="/data/select_multiple" type="string"/>
      <bind calculate="concat('uuid:', uuid())" nodeset="/data/meta/instanceID" readonly="true()" type="string"/>
    </model>
  </h:head>
  <h:body>
    <select ref="/data/select_multiple">
      <itemset nodeset="randomize(instance('choices')/root/item)">
        <value ref="name"/>
        <label ref="label"/>
      </itemset>
    </select>
  </h:body>
</h:html>
