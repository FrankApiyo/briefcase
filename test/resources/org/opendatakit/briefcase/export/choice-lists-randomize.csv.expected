SubmissionDate,select_multiple,select_multiple/a,select_multiple/b,select_multiple/c,select_multiple/d,select_multiple/e,meta-instanceID,KEY
"Apr 26, 2020 8:58:20 AM",a,1,0,0,0,0,uuid:39f3dd36-161e-47cb-a1a4-395831d253a7,uuid:39f3dd36-161e-47cb-a1a4-395831d253a7
